rockPaperScissors.py

```diff
@@ -3,48 +3,51 @@
 
 import random
 
-def gameLogic(player, computer, playerScore, computerScore):
-    playerIndex = RPS.index(player)
-    computerIndex = RPS.index(computer)
-
-    gameIncrement = 0
-    if (playerIndex != computerIndex):
-        gameIncrement = 1
-        playerNextIndex = (playerIndex + 1) % len(RPS)
-        
-        if (playerNextIndex == computerIndex):
-            computerScore += 1
-        else:
-            playerScore += 1
+def gameLogic(player1Choice, player2Choice):
+    if player1Choice == player2Choice:
+        return 0
 
-    return gameIncrement, playerScore, computerScore
+    playerNextIndex = (player1Choice + 1) % 3
```

Comment:
this was a pretty slick suggestion your friend made. It works great - at least until we start playing rock paper scissors lizard spock. Then things get real interesting...

Reply:
does anyone actually play that game?

Reply:
If not it’s lnly because no one has yet proven themselves man enough to write the program to let them play it